widget-missing-in-kendoui-professional-package.md

```diff
@@ -0,0 +1,97 @@
+
+---
+title: Widgets from the Kendo UI Core/Professional Package Don't Load
+description: After installing Kendo UI via the Package Manager, widget XXXX doesn't work
```

Comment:
widget XXXX should be "one or more widgets no longer work" unless you can list the widgets.